internal/logic/logic.go

```diff
@@ -203,7 +202,6 @@ func (l userMgmtSvcLogic) Login(w http.ResponseWriter, credential model.LoginCre
 }
 
 func (l userMgmtSvcLogic) Activate(id any) *respModel.Response {
```

Comment:
Why is any used as the argument? This should be a string.

Reply:
done